CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ For details about compatibility between different releases, see the **Commitment
 
 ### Changed
 
+- Stack is now built with Go 1.21.
```

Comment:
And CLI right? Maybe explicitly mention the binaries? Or something funky like`tt{n|i}-lw-{stack|cli}`.

Reply:
Should've been `The Things Stack`, which is probably clearer.